src/OpenTelemetry/Resources/OtelEnvResourceDetector.cs

```diff
@@ -38,15 +39,74 @@ private static List<KeyValuePair<string, object>> ParseResourceAttributes(string
         string[] rawAttributes = resourceAttributes.Split(AttributeListSplitter);
```

Comment:
These `string[]` and `string` allocations could be removed with span operations.

Reply:
Are you saying this is the only place in the whole codebase that splits strings and shouldn't?

Polyfills can be easily created for targets that miss some features. I've done that more than once.